example/centos/centos-9-x86_64-ami.yaml

```diff
@@ -1,37 +1,139 @@
+                minpoll: 4
+                maxpoll: 4
+                iburst: true
+                prefer: true
+            leapsectz: ""
+        - type: org.osbuild.sysconfig
+          options:
+            kernel:
+              update_default: true
+              default_kernel: kernel
+            network:
+              networking: true
+              no_zero_conf: true
+            network-scripts:
+              ifcfg:
+                eth0:
+                  bootproto: dhcp
+                  device: eth0
+                  ipv6init: false
+                  onboot: true
+                  peerdns: true
+                  type: Ethernet
+                  userctl: true
+        - type: org.osbuild.systemd-logind
+          options:
+            filename: 00-getty-fixes.conf
+            config:
+              Login:
+                NAutoVTs: 0
+        - type: org.osbuild.cloud-init
+          options:
+            filename: 00-rhel-default-user.cfg
+            config:
+              system_info:
+                default_user:
+                  name: ec2-user
+        - type: org.osbuild.modprobe
+          options:
+            filename: blacklist-nouveau.conf
+            commands:
+              - command: blacklist
+                modulename: nouveau
+        - type: org.osbuild.modprobe
+          options:
+            filename: blacklist-amdgpu.conf
+            commands:
+              - command: blacklist
+                modulename: amdgpu
+        - type: org.osbuild.dracut.conf
+          options:
+            filename: sgdisk.conf
+            config:
+              install_items:
+                - sgdisk
+        - type: org.osbuild.dracut.conf
+          options:
+            filename: ec2.conf
+            config:
+              add_drivers:
+                - nvme
+                - xen-blkfront
+        - type: org.osbuild.systemd.unit
+          options:
+            unit: nm-cloud-setup.service
+            dropin: 10-rh-enable-for-ec2.conf
+            config:
+              Service:
+                Environment:
+                  - key: NM_CLOUD_SETUP_EC2
+                    value: "yes"
+        - type: org.osbuild.authselect
+          options:
+            profile: sssd
+        - type: org.osbuild.sshd.config
+          options:
+            config:
+              PasswordAuthentication: false
+        - otk.external.otk-make-fstab-stage:
+            ${filesystem}
+        - otk.include: "fragment/grub2.yaml"
+        - type: org.osbuild.systemd
+          options:
+            enabled_services:
+              - sshd
+              - NetworkManager
+              - nm-cloud-setup.service
+              - nm-cloud-setup.timer
+              - cloud-init
+              - cloud-init-local
+              - cloud-config
+              - cloud-final
+              - reboot.target
+              - tuned
+            default_target: multi-user.target
+        - type: org.osbuild.selinux
```

Comment:
this should be split into a fragement as it's use with the same options everywhere